src/app/modules/gb-data-selection-module/gb-data-selection.component.html

```diff
@@ -39,8 +39,8 @@
       <gb-selection></gb-selection>
     </md2-accordion-tab>
 
-    <!-- The 2nd step -->
-    <md2-accordion-tab [active]="false">
+    <!-- The 2nd step: variable selection -->
+    <md2-accordion-tab [active]="true">
```

Comment:
is it a change on purpose (second step should be active by default) or was added just for testing?

Reply:
sorry, just added for testing, will change it back